code-reviews/README.md

```diff
@@ -18,6 +18,80 @@ Code reviews is a way to have a conversation about the code where participants w
 - [FAQ](./faq.md)
 - [Resources](#resources)
 
+### Summary of contents
+
+- [Process Guidance: General](./process-guidance/README.md#general-guidance)
+  - [Pull Requests](./pull-requests.md)
+    - [General Process](./pull-requests.md#general-process)
+    - [Size Guidance](./pull-requests.md#size-guidance)
+    - [Commit Message Standardization](./pull-requests.md#commit-message-standardization)
+    - [Pull Request Template](./pull-request-template.md)
```

Comment:
I suggested to move commit message standartization to Source code section. We need to discuss this - I can do it in another PR though